app/components/pages/SubmissionWizard/steps/Editors/EditorsPage.js

```diff
@@ -275,29 +275,31 @@ class EditorsPage extends React.Component {
 
           {values.suggestedReviewers.map((_, index) => (
             // eslint-disable-next-line react/no-array-index-key
-            <Box key={index} mb={2}>
-              <TwoColumnLayout bottomSpacing={false}>
-                <ValidatedField
-                  label={
-                    index < limits.suggestedReviewers.min
-                      ? `Reviewer ${index + 1} name`
-                      : `Reviewer ${index + 1} name (optional)`
-                  }
-                  name={`suggestedReviewers.${index}.name`}
-                  onChange={this.handleSuggestedReviewersChanged}
-                />
-                <ValidatedField
-                  label={
-                    index < limits.suggestedReviewers.min
-                      ? `Reviewer ${index + 1} email`
-                      : `Reviewer ${index + 1} email (optional)`
-                  }
-                  name={`suggestedReviewers.${index}.email`}
-                  onChange={this.handleSuggestedReviewersChanged}
-                  type="email"
-                />
-              </TwoColumnLayout>
-            </Box>
+            <div data-test-id="suggestedReviewerInputGroup" key={index}>
```

Comment:
Is there a reason not to use `React.Fragment` instead of `div` here?

Reply:
However I suppose I could have an inner `React.Fragment` then run the selector as `wrapper.find('[data-test-id="suggestedReviewerInputGroup"] > React.Fragment')` that'd remove the need for the unnecessary `div`.

Reply:
Managed to get the selector working without having to add a `div` or `React.Fragment` after reviewing. Thanks!

Reply:
Ok sounds reasonable, thank you 👍